app-crypt/acme-sh/acme-sh-2.7.9.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-/.}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="An ACME Shell script"
+HOMEPAGE="https://github.com/Neilpang/acme.sh"
+SRC_URI="https://github.com/Neilpang/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libressl"
```

Comment:
Ok, I'm really sorry that I've missed this earlier. Since this is a shell script and you don't hardcode anything based on this flag's state, it should be entirely possible to swap libressl and openssl at runtime with no consequences. Therefore, please remove the flag and simply use `|| ( ... )` dep between the two.

Reply:
Flag removed.